KPCTabsControl/DefaultTheme.swift

```diff
@@ -13,7 +13,6 @@ import Cocoa
  */
 public struct DefaultTheme: Theme {
 
-    // TODO: Why is this necessary?
```

Comment:
Without an empty initializer, it works from the same target; but client code will not have access to this "implicit" initializer by default.